apps/services/license-api/src/app/modules/license/clients/disabilityLicense/disabilityLicenseClient.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { GenericLicenseClient } from '../../license.types'
+import type { Logger } from '@island.is/logging'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import {
+  Pass,
+  PassDataInput,
+  Result,
+  RevokePassData,
+  SmartSolutionsApi,
+  VerifyPassData,
+} from '@island.is/clients/smartsolutions'
+
+@Injectable()
+export class DisabilityLicenseClientService implements GenericLicenseClient {
+  constructor(
+    @Inject(LOGGER_PROVIDER) private logger: Logger,
+    private smartApi: SmartSolutionsApi,
+  ) {}
+
+  pushUpdate(
+    inputData: PassDataInput,
+    nationalId: string,
+  ): Promise<Result<Pass | undefined>> {
+    this.logger.debug('in push update for Disability license')
+    return this.smartApi.updatePkPass(inputData, nationalId)
+  }
+
+  async pullUpdate(): Promise<Result<Pass | undefined>> {
+    this.logger.debug('in pull update for Disability license')
+    return {
+      ok: false,
+      error: {
+        code: 99,
+        message: 'not implemented yet',
+      },
+    }
+  }
```

Comment:
Why not throw an error? 
```suggestion
    throw new Error('Not yet implemented')
  }
```

Reply:
good point